docs/lab/gce/debian-driver.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: kmm.sigs.k8s.io/v1beta1
+kind: Module
+metadata:
+  name: kmm-ci-deb-a
+spec:
+  moduleLoader:
+    container:
+      modprobe:
+        moduleName: kmm-ci-deb-a 
+
+      kernelMappings:
+        - literal: 5.10.127 
+          containerImage: quay.io/ebelarte/kmmo:gke-kmm-deb-1 
+          build:
+            buildArgs:
+              - name: CI_BUILD_ARG
+                value: some-build-arg
+            dockerfile: |
+              FROM debian:stable-slim as builder
+              ARG KERNEL_VERSION=''
+              RUN apt-get update && apt-get install -y \ 
+                  build-essential \
+                  git \
+                  bc \
+                  wget \
+                  flex \
+                  bison \
+                  libssl-dev \
+                  libelf-dev \
+                  python3 
+                  #bison \
+                  #flex \
+                  #libelf-dev \
+                  #gnupg \
+                  #wget \
+                  #git \
+                  #make \
+                  #bc 
+                  #gcc
+ 
+              RUN kv=${1:-v"$(uname -r | sed -E 's/\+*$//')"} && \
+                  wget "https://chromium.googlesource.com/chromiumos/third_party/kernel/+archive/$kv.tar.gz" && \
+                  mkdir -p /src/kernel && \
+                  tar xzf "$kv.tar.gz" -C /src/kernel
+              WORKDIR /src/kernel
+              RUN gunzip < /proc/config.gz > .config
+              RUN yes "" | make oldconfig && make prepare && make scripts
+              RUN git clone https://github.com/kubernetes-sigs/kernel-module-management.git && \
+                  mv kernel-module-management/ci/kmm-kmod kmm-kmod
+             
+              WORKDIR kmm-kmod
+              RUN make -C /src/kernel
+
+    
+              FROM debian:stable-slim 
+              ARG KERNEL_VERSION
+              RUN apt-get update && apt-get install kmod
+              COPY --from=builder /src/kernel/kmm-kmod/kmm_ci_a.ko /opt/lib/modules/${KERNEL_VERSION}/
+              COPY --from=builder /src/kernel/kmm-kmod/kmm_ci_b.ko /opt/lib/modules/${KERNEL_VERSION}/
+              RUN depmod -b /opt
+  imageRepoSecret: 
+    name: quay-eb 
```

Comment:
Can we use a generic secret name? also adding `<>` will help the user understand it needs to modify it, for example `<my-secret>`